lib/logstash/outputs/scalyr.rb

```diff
@@ -292,6 +292,10 @@ def multi_receive(events)
 
       while !multi_event_request_array.to_a.empty?
         multi_event_request = multi_event_request_array.pop
+        exc_data = nil
```

Comment:
Comment on what those variables hold and how are they used may be useful. Besides that, LGTM.